tests/test_octodns_provider_cloudflare.py

```diff
@@ -2074,3 +2078,32 @@ def test_idna_domain(self):
             provider.populate(zone)
         self.assertEqual(8, len(zone.records))
         self.assertEqual(zone.name, idna_encode('gíthub.com.'))
+
+    def test_account_id_filter(self):
+        provider = CloudflareProvider(
+            'test',
+            'email',
+            'token',
+            account_id='334234243423aaabb334342aaa343433',
```

Comment:
This will make the unsupported DNAME record a warning rather than exception since it's not relevant to the stuff being tested here. 

```suggestion
            account_id='334234243423aaabb334342aaa343433',
            strict_supports=False,
```